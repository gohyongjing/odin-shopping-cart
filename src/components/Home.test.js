import React from 'react';
import { render } from "@testing-library/react";
import Home from './Home';

describe("Home component", () => {
    it("renders", () => {
        const { asFragment } = render(<Home />);
        expect(asFragment()).toMatchSnapshot();
    });

    it
});
