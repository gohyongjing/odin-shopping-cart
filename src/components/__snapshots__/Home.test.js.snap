// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders 1`] = `
<DocumentFragment>
  <div
    class="home"
  >
    <h1>
      Welcome to the Shop!
    </h1>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    </p>
    <p>
      Proin mattis sapien ut semper aliquam. Sed non enim eget sapien euismod cursus. Aliquam accumsan tristique dolor, ut sagittis sapien volutpat quis. Curabitur pellentesque nec magna aliquam finibus. Nulla euismod sit amet purus nec laoreet. Mauris vel ullamcorper ligula, non molestie odio. Mauris vitae blandit velit. Nam vitae dui efficitur, ullamcorper diam vitae, molestie nisi. Morbi placerat est sed est fermentum faucibus. Morbi sagittis felis sit amet sapien gravida fermentum. Maecenas ut mauris sodales, pulvinar lectus vitae, efficitur lacus. Cras turpis nulla, vestibulum sit amet feugiat blandit, malesuada aliquam erat. Nulla vulputate massa augue, eget porttitor arcu suscipit ac. Phasellus sed ultricies orci, eget sagittis urna. Maecenas ac felis non ligula eleifend vehicula at quis nulla. Proin facilisis vestibulum erat, et ultricies nisl ornare a.
    </p>
  </div>
</DocumentFragment>
`;
